src/components/CartItem/index.js

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button/styled';
+import StyledProductName from '../ProductName/styled';
+import StyledProductPrice from '../ProductPrice/styled';
+
+import StyledCartItem from './styled';
+
+export default function CartItem({productid, id, imgUrl, name, price}) {
```

Comment:
Just one typo for `productid`. If camelCase is followed, it should be `productId` :)